Xamarin.Forms.Core/Color.cs

```diff
@@ -114,6 +114,26 @@ public Color(double r, double g, double b, double a) : this(r, g, b, a, Mode.Rgb
 			}
 		}
 
+		Color(byte r, byte g, byte b)
+		{
+			_mode = Mode.Rgb;
+			_r = r / 255f;
+			_g = g / 255f;
+			_b = b / 255f;
+			_a = 1;
+			ConvertToHsl(_r, _g, _b, _mode, out _hue, out _saturation, out _luminosity);
+		}
+
+		Color(byte r, byte g, byte b, byte a)
+		{
+			_mode = Mode.Rgb;
+			_r = r / 255f;
+			_g = g / 255f;
+			_b = b / 255f;
+			_a = a / 255f;
+			ConvertToHsl(_r, _g, _b, _mode, out _hue, out _saturation, out _luminosity);
+		}
```

Comment:
@StephaneDelcroix these new constructors here performed better than precomputing the `float` values: https://github.com/jonathanpeppers/Xamarin.Forms/commit/607b977cc86858e4fccd68575febdb423fa83102

It seems like the huge list of `float` always made things worse.